docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -1,106 +1,106 @@
-. Save your *Identity anchor*.
+At this point, depending on the device you are using, you might be asked to either use your device authenitication method, or to use an identity saved to your security key. If you are registering for the first time, choose to use the device authentication.
+. Save your Identity Anchor.
 +
-When you have registered your device, you’ll receive a *Identity anchor*. 
+When your device has been added, you’ll receive an Identity Anchor.
 +
-This number is unique, but it is not a secret, so you should save the number in multiple places. 
-Your browser will remember your identity anchor, but you will need it if you log in on a different computer, change your browser profile, or if you clear your browser state. 
+Your Identity Anchor is represented by a unique number. It is not a secret and you should save it in multiple places so you don't lose it.
+Your browser will remember your Identity Anchor, but you will need it if you authenticate on a different computer, change your browser profile, or if you clear your browser state.
 +
-WARNING: If you lose your identity anchor and are logged out on all devices, you cannot log in to Internet Identity to manage your devices, or access any of your applications, unless you have set up account recovery in the next step.
+WARNING: If you lose your Identity Anchor and are logged out of all devices, you can no longer authenticate to Internet Identity to manage your devices, or access any of your applications, unless you have set up account recovery using a seed phrase in the next step. So don't lose your Identity Anchor!
 
 . Click *Continue*. 
 
-. Set up account recovery
+. Add a recovery mechanism to an Identity Anchor
 +
-In addition to registering multiple devices and using security keys, you can set up account recovery at the prompt by clicking *Set up Account recovery*. 
+In addition to adding multiple devices and using security keys, you can set up account recovery at the prompt by clicking *Add a recovery mechanism to an Identity Anchor*.
 +
 On the next screen, you can select one of the following options:
 
 * *Seed Phrase*
 + 
-Select this option to generate a cryptographically-secure seed-phrase that you can use to recover your account. Make sure you save this phrase somewhere safe and known only to you. *Note that the first string in your seed phrase is your identity anchor*. You will need this number to begin the the recovery process.
+Select this option to generate a cryptographically-secure seed phrase that you can use to recover an Identity Anchor. Make sure you save this phrase somewhere safe and known only to you, since anyone who knows the seed phrase will be able to take full control of this Identity Anchor. *Note that the first string in your seed phrase is the Identity Anchor*. You will need this number to begin the the recovery process.
```

Comment:
```suggestion
Select this option to generate a cryptographically-secure seed phrase that you can use to recover an Identity Anchor. Make sure you save this phrase somewhere safe and known only to you, since anyone who knows the seed phrase will be able to take full control of this Identity Anchor. *Note that the first string in your seed phrase is the Identity Anchor*. You will need this number to begin the recovery process.
```

Reply:
Fixed.